quran/quran.com-frontend-next
# Fixed SVG icons in Safari

### Summary
Closes #661. SVG icons were broken on Safari as it seems that Safari requires SVGs in`flex` wrappers to have width/height attributes.

### Test Plan
- Run master's dev server.
- Open localhost on Safari.
- Open the dropdown icon for verse actions.
- You should see this:
![139581453-68dd0e6a-985e-46ff-adee-f16f6fef01cf](https://user-images.githubusercontent.com/3989750/140662768-3f342f5f-3705-4f9f-8cb8-3a8057762571.png)

- Switch to this branch and check the same menu, it should be fixed now:
![image](https://user-images.githubusercontent.com/3989750/140662785-a1809814-ce61-4d72-8767-633fade1a1d3.png)


### Screenshots
above ^
